src/tcgwars/logic/impl/gen3/RubySapphireNG.groovy

```diff
@@ -432,15 +500,38 @@ public enum RubySapphireNG implements LogicCardInfo {
         weakness F
         pokeBody "Lazy", {
           text "As long as Slaking is your Active Pokémon, your opponent's Pokémon can't use any Poké-Powers."
-          delayedA {
+          getterA IS_ABILITY_BLOCKED, { Holder h->
+            if (self.active && h.effect.target.owner == self.owner.opposite && h.effect.ability instanceof PokePower) {
+              h.object=true
+            }
+          }
+          //TODO: Is this needed here?
```

Comment:
Currently yes. This is the getter that `checkGlobalAbility()` checks when you want to be able to block it. It may be possible to use the normal `IS_ABILITY_BLOCKED` getter instead there though, but that wouldn't work with things like `instanceof PokePower` and is something we probably need to add a way of checking to `IS_GLOBAL_ABILITY_BLOCKED` too.

Reply:
👍 

(another moved-over card, but thanks for responding to it anyway 😅 )

Reply:
Figured it was just a general copy/pasted thing, not specific to this PR or even this Card. Figured I'd share some knowledge there in case it was relevant to whoever added the TODO in the first place. It'll get to whoever wrote it in the first place eventually.

Reply:
~~Yeah, wonder who wrote that...~~